flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/test/java/org/apache/flink/stormcompatibility/api/FlinkTestCluster.java

```diff
@@ -25,15 +25,21 @@
 import backtype.storm.generated.StormTopology;
 import backtype.storm.generated.SubmitOptions;
 import backtype.storm.generated.TopologyInfo;
+
+import org.apache.flink.api.common.JobID;
+import org.apache.flink.runtime.jobgraph.JobGraph;
 import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
 import org.apache.flink.streaming.util.TestStreamEnvironment;
 
+import java.util.HashMap;
 import java.util.Map;
 
 /**
  * {@link FlinkTestCluster} mimics a Storm {@link LocalCluster} for ITCases via a {@link TestStreamEnvironment}.
  */
 public class FlinkTestCluster extends FlinkLocalCluster {
+	// TODO remove and look up job
```

Comment:
This is also a larger issue in Storm compatibility (and not related to this PR). Need to add helper method to `ClusterUtil` to get running jobs from `LocalFlinkMiniCluster` which is used under the hood. I would rather have another JIRA for this than including it here. Effects `FlinkLocalCluster`, too.